circuits/cpp/src/aztec3/circuits/kernel/public/common.hpp

```diff
@@ -193,9 +193,8 @@ void common_validate_inputs(DummyComposer& composer, KernelInput const& public_k
  * @param public_kernel_inputs The inputs to this iteration of the kernel circuit
  * @param circuit_outputs The circuit outputs to be populated
  */
-template <typename KernelInput>
-void propagate_valid_state_transitions(KernelInput const& public_kernel_inputs,
-                                       KernelCircuitPublicInputs<NT>& circuit_outputs)
+template <typename KernelInput> void propagate_valid_state_transitions(KernelInput const& public_kernel_inputs,
```

Comment:
Again, not sure I like this moving down to be on same line as function declaration. Seems like in some files it does this, in some files it leaves it.

Reply:
That doesn't seem right. Maybe I have the settings different for functions/structs/enums/classes. I'll look into it.